src/debugpy/_vendored/pydevd/pydevd_plugins/django_debug.py

```diff
@@ -427,7 +427,7 @@ def __init__(self, frame, original_filename, lineno, f_locals):
         self.f_trace = None
 
 
-def change_variable(frame, attr, expression, default):
+def change_variable(frame, attr, expression, default, scope=None):
     if isinstance(frame, DjangoTemplateFrame):
         result = eval(expression, frame.f_globals, frame.f_locals)
         frame._change_variable(attr, result)
```

Comment:
I don't know what these plug-ins are doing, but should they be passing on the `scope` value?

Reply:
Yeah I suppose they should. That one is using the code I wrote.